app/View/Pages/home.ctp

```diff
@@ -58,19 +58,19 @@ endif;
 	$settings = Cache::settings();
 	if (!empty($settings)):
 		echo '<span class="notice success">';
-			echo __d('cake_dev', 'The %s is being used for core caching. To change the config edit %s', '<em>' . $settings['engine'] . 'Engine</em>', 'APP/Config/core.php');
+			echo __d('cake_dev', 'The %s is being used for core caching. To change the config edit %s', '<em>' . $settings['engine'] . 'Engine</em>', 'CONFIG/core.php');
 		echo '</span>';
 	else:
 		echo '<span class="notice">';
-			echo __d('cake_dev', 'Your cache is NOT working. Please check the settings in %s', 'APP/Config/core.php');
+			echo __d('cake_dev', 'Your cache is NOT working. Please check the settings in %s', 'CONFIG/core.php');
```

Comment:
Same as in https://github.com/cakephp/cakephp/pull/10579/files#diff-17d0cb9c5cd4a62e574aa3dc8083249eR61